lib/pharos/scripts/configure-etcd.sh

```diff
@@ -78,8 +65,19 @@ spec:
       type: DirectoryOrCreate
     name: etcd-certs
 EOF
-systemctl daemon-reload
-systemctl restart kubelet
+
+if [ ! -e /etc/kubernetes/kubelet.conf ]; then
```

Comment:
Good, no more unnecessary kubelet restarts, but now we're back to the case where `pharos-cluster up` won't work after a `kubeadm reset`... however, that's probably okay. Using `kubeadm reset && apt-get purge kubelet kubeadm kubectl` is enough for testing (although that leaves behind some `/etc/systemd/system/kubelet.service.d/*.conf` bits, but that's okay as long as the `5-pharos-etcd.conf` here overrides them).

This condition can probably be changed back to the original `if [ ! -e /usr/bin/kubelet ]`, which makes more sense here. I don't think we can reasonably expect this to behave correctly in all possible kubelet configuration states because of the extra complexity in the two-stage kubelet configuration.

Reply:
Makes more sense to write this as it used to be: `if [ ! -e /usr/bin/kubelet ]`